pkg/test/framework/components/environment/kube/flags.go

```diff
@@ -209,6 +209,8 @@ func init() {
 		"A comma-separated list of paths to kube config files for cluster environments (default is current kube context)")
 	flag.BoolVar(&settingsFromCommandLine.Minikube, "istio.test.kube.minikube", settingsFromCommandLine.Minikube,
 		"Indicates that the target environment is Minikube. Used by Ingress component to obtain the right IP address..")
+	flag.BoolVar(&settingsFromCommandLine.CentralIstiod, "istio.test.kube.centralIstiod", settingsFromCommandLine.Minikube,
```

Comment:
this shouldn't be a flag in my opinion. That is for global settings and would require us to do a whole new job just for central istiod. It should just be a configuration on the test, like the other control plane stuff

Reply:
removed